conda-forge.yml

```diff
@@ -6,16 +6,16 @@ build_platform:
   linux_aarch64: linux_64
   linux_ppc64le: linux_64
 conda_build:
+  error_overlinking: true
```

Comment:
This will check that `cudart` was statically linked (as we don't have the dynamic library as a dependency meaning it will raise an error otherwise)